apollo-portal/src/test/java/com/ctrip/framework/apollo/portal/service/NamespaceServiceTest.java

```diff
@@ -223,6 +224,43 @@ public void testDeleteEmptyNamespace() {
 
   }
 
+  @Test
+  public void testLoadNamespaceBO() {
+    String branchName = "branch";
+    NamespaceDTO namespaceDTO = createNamespace(testAppId, branchName, testNamespaceName);
+    when(namespaceAPI.loadNamespace(any(), any(), any(), any())).thenReturn(namespaceDTO);
+
+    ReleaseDTO releaseDTO = new ReleaseDTO();
+    releaseDTO.setConfigurations("{\"k1\":\"k1\",\"k2\":\"k2\", \"k3\":\"\"}");
+    when(releaseService.loadLatestRelease(any(), any(), any(), any())).thenReturn(releaseDTO);
+
+    List<ItemDTO> itemDTOList = Lists.newArrayList();
+    ItemDTO itemDTO1 = new ItemDTO();
+    itemDTO1.setId(1);
+    itemDTO1.setNamespaceId(1);
+    itemDTO1.setKey("k1");
+    itemDTO1.setValue(String.valueOf(1));
+    itemDTOList.add(itemDTO1);
+
+    ItemDTO itemDTO2 = new ItemDTO();
+    itemDTO2.setId(1);
```

Comment:
```suggestion
    itemDTO2.setId(2);
```
I think use `itemDTO2.setId(2);` is better. What do you think?

Reply:
It's done